composer.json

```diff
@@ -26,6 +26,7 @@
         "symfony/framework-bundle": "~2.3"
     },
     "require-dev": {
+        "friendsofsymfony/http-cache": "~1.2@dev",
```

Comment:
``@dev`` in requirements is root-only. It is ignored in dependencies. Only the root package can control stability restrictions